docs/users-guide/09-multi-series-charting.md

```diff
@@ -7,73 +7,79 @@ Data in isolation is rarely all that useful. One of the best ways to add context
 - New users per day vs. returning users per day.
 - Orders per day from a few different product lines.
 
-### In Metabase there are two main ways to get data side-by-side.
+### Two ways to display data side-by-side
 
-1. Combining two existing saved questions that share a common dimension (like time) on a dashboard
+1. [**Use the notebook editor to ask a question that involves multiple dimensions**](#using-the-notebook-editor-to-create-a-multi-series-visualization) (or in SQL, if you’re fancy). Example: the count of users by region over time.
 
-e.g. Let me see revenue over time and cost over time together.
+2. [**Combine two saved questions**](#combining-two-saved-questions) that share a common dimension (like time) on a dashboard. For example, let me see revenue over time and cost over time together.
 
-2. Asking a question that involves multiple dimensions in the query builder (or in SQL if you’re fancy).
+### Using the notebook editor to create a multi-series visualization
 
-e.g. The count of users by region over time.
+If you’re creating a new question, you can view the results as a multi-series visualization by summarizing your data and grouping it into two or more groups.
```

Comment:
I think it has to be _exactly_ two groups. If you try breaking out the count of orders by Created_At, Product.Category, _and_ People.State, you'll have to pick which two of the breakouts to use for the x-axis and for the series breakout; the third breakout won't be used.